fe/src/main/java/org/apache/doris/catalog/PrimitiveType.java

```diff
@@ -56,6 +56,7 @@
     TIME("TIME", 8, TPrimitiveType.TIME),
     // we use OBJECT type represent BITMAP type in Backend
     BITMAP("BITMAP", 16, TPrimitiveType.OBJECT),
+    ARRAY("ARRAY", 16, TPrimitiveType.ARRAY),
```

Comment:
```suggestion
    ARRAY("ARRAY", 17, TPrimitiveType.ARRAY),
```

Reply:
The slot_size is used for Tuple to compute memory layout, why need change to 17?

Reply:
The slot_size is used for Tuple to compute memory layout, why need change to 17?

Reply:
Oh, I misunderstood..